packages/runtime/src/helpers/redirects.ts

```diff
@@ -25,14 +25,6 @@ import {
 const matchesMiddleware = (middleware: Array<string>, route: string): boolean =>
   middleware.some((middlewarePath) => route.startsWith(middlewarePath))
 
-const generateHiddenPathRedirects = ({ basePath }: Pick<NextConfig, 'basePath'>): NetlifyConfig['redirects'] =>
-  HIDDEN_PATHS.map((path) => ({
```

Comment:
Not needed anymore, as these aren't published

Reply:
What are/were `HIDDEN_PATHS`?

Reply:
They were the files in `.next` that we didn't want to be publicly-visible in the deploy, such as `cache`, `server` and various json files